roles/StackStorm.rabbitmq/defaults/main.yml

```diff
@@ -3,3 +3,5 @@ rabbitmq_plugins: []
 # To enable the management plugin (in which case you'd want at least one user tagged with administrator):
 #rabbitmq_plugins:
 #  - rabbitmq_management
+# Set to "latest" to latest version, or specify specific version
+rabbitmq_version: "latest"
```

Comment:
I think what @Kami wanted to say is that expectations from the 3rd party software that stackstorm relies on is to use `present` which will install the version once and won't update it automatically as it happens with the `latest`.

Looks like we're following the same with the `mongodb`, `nginx` and `nodejs` roles.

Reply:
Aha - makes sense - I'll update.

Reply:
@armab  Have updated variable to use "present", it was using present in the tasks - but name could have been confusing - so went back to naming it "present".
Let me know if that isn't what we wanted.